### PR TITLE
Added metric plugin for fluentd monitor_agent

### PR DESCRIPTION
**Pull request by jun06t**

### CODE REVIEW

File: bin/metric-fluentd-monitor-agent.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+class MetricFluentdMonitorAgent < Sensu::Plugin::Metric::CLI::Graphite
+  option :scheme,
+         description: 'Metric naming scheme',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.fluentd"
+  option :url,
+         short: '-u URL',
+         long: '--url URL',
+         description: 'A URL to connect to',
```
Comment (analytically) [inline bin/metric-fluentd-monitor-agent.rb]: Mind rewording this? It's the fluentd plugins json path I guess...

File: bin/metric-fluentd-monitor-agent.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env ruby
```
Comment (analytically) [inline bin/metric-fluentd-monitor-agent.rb]: No space.

File: bin/metric-fluentd-monitor-agent.rb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#
+#   metric-fluentd-monitor
+#
+# DESCRIPTION:
+#   This plugin uses fluentd monitor_agent to monitor buffer, produces
+#   Graphite formated output.
```
Comment (analytically) [inline bin/metric-fluentd-monitor-agent.rb]: Typo.

File: bin/metric-fluentd-monitor-agent.rb [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+         proc: proc(&:to_i)
+
+  def run
+    if config[:url]
+      uri = URI.parse(config[:url])
+      config[:host] = uri.host
+      config[:port] = uri.port
+      config[:request_uri] = uri.request_uri
+    else
+      unknown 'No URL specified'
```
Comment (analytically) [inline bin/metric-fluentd-monitor-agent.rb]: How can this be the case when there's a default?

**Comment by jun06t** [comment]
@analytically 
Thanks for your review. I've fixed it.

**Pull request merged**